src/Components/NavBar/Menus/MobileNavMenu.tsx

```diff
@@ -15,8 +12,12 @@ import {
   Serif,
 } from "@artsy/palette"
 
+import { SystemContext } from "Artsy"
+import { useTracking } from "Artsy/Analytics/useTracking"
+import * as auth from "Components/NavBar/Utils/auth"
```

Comment:
Off-topic: This should be renamed to authentication, or at the very least authn (although I personally prefer actual words).